lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -321,6 +321,7 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
                 RowwiseQuantizedSparseLengthsWeightedSumNode::LengthsIdx) ==
             ElemKind::Int32ITy);
 
+  case Kinded::Kind::EmbeddingBagByteRowwiseOffsetsNodeKind:
```

Comment:
Do the two here support all the same data types? Even if they do it's at best confusing to read this code seeing that the EmbeddingBag will check inputs and outputs types based on RWQSLWS's indices for inputs and outputs. It's fragile in case ordering of inputs and outputs ever deviates, even if that is pretty unlikely.

Reply:
Yep this was a bad idea, I'll fix it